.circleci/config.yml

```diff
@@ -1,4 +1,6 @@
 version: 2.1
+orbs:
```

Comment:
can you change it back to use `make shellcheck`?

we actually moved away from this orb -- it was important to us that the local Makefile and the circleci job ran the same code

https://github.com/tilt-dev/tilt-extensions/pull/223

you'll need to add a setup_remote_docker to make the docker commands work
https://circleci.com/docs/building-docker-images/

Reply:
done. ran into other issues (volume mounts work different than expected). Found different solution. Will explain that in a further comment.